docs/usage/contrib/sqlalchemy.rst

```diff
@@ -6,7 +6,7 @@ Litestar comes with a built-in repository class (:class:`SQLAlchemyAsyncReposito
 Features
 --------
 
-* Pre-configured ``DeclarativeBase`` for `SQLAlchemy <https://docs.sqlalchemy.org/>`_ 2.0 that includes a UUID based primary-key and an optional version with audit columns.
+* Pre-configured ``DeclarativeBase`` for `SQLAlchemy <https://docs.sqlalchemy.org/>`_ 2.0 that includes a UUID based primary-key, a  `sentinel column <https://docs.sqlalchemy.org/en/20/core/connections.html#configuring-sentinel-columns/>`_ and an optional version with audit columns.
```

Comment:
I think this can be replaced because of `intersphinx`. I feel like the connections block could be too but I can't figure it out... @provinzkraut may know how
```suggestion
* Pre-configured ``DeclarativeBase`` for :doc:`SQLAlchemy <sqlalchemy:index>` 2.0 that includes a UUID based primary-key, a  `sentinel column <https://docs.sqlalchemy.org/en/20/core/connections.html#configuring-sentinel-columns/>`_ and an optional version with audit columns.
```

Reply:
Is this ready to merge as is?  I can accept the suggestion if so.  I think this is the only outstanding task on this PR.